e2e/sync_test.go

```diff
@@ -852,6 +856,235 @@ func TestSyncS3BucketToS3BucketSameSizesSourceOlder(t *testing.T) {
 	}
 }
 
+// sync s3://bucket/* s3://destbucket/ (source is glacier, destination is standard)
+func TestSyncS3BucketToS3BucketSourceGlacierDestinationStandard(t *testing.T) {
+	t.Parallel()
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	dstbucket := s3BucketFromTestNameWithPrefix(t, "dst")
+
+	createBucket(t, s3client, bucket)
+	createBucket(t, s3client, dstbucket)
+
+	// put objects in source bucket
+	S3Content := map[string]string{
```

Comment:
```suggestion
	s3Content := map[string]string{
```

Reply:
It has been taken from older tests. Should I change those too ?

Reply:
I think we should. Variables should be camelcase

Reply:
done.